### ISSUE TITLE
Fix Appveyor

### ISSUE THREAD

**Issue by claireh93** [ScottishCovidResponse/SCRCIssueTracking#350]
At the moment allowed to fail, but needs fixing in future - @richardreeve can provide details.

**Comment by jwscook** [comment]
@richardreeve Could you provide the details here please?

**Comment by richardreeve** [comment]
The latest PR merge fails in appveyor testing here - https://ci.appveyor.com/project/richardreeve/simulation-jl/builds/33406532/job/bh0g7t849pfu4ies#L383 - with a failure to build ORCA. I notice that I may have been wrong about it generally failing - it may just be x86 and not x64, in which case we may just be able to drop 32-bit Windows?

**Comment by claireh93** [comment]
Is this still an issue @richardreeve or can we close?

**Comment by richardreeve** [comment]
No, sadly appveyor still fails... we're just looking for someone with a Windows machine to try to work out what's going on. Current status is here - https://ci.appveyor.com/project/richardreeve/simulation-jl - and it seems to fail everything.
